systemd/ceph-crash.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Ceph crash dump collector
+After=ceph-mgr.target
```

Comment:
I would leave this off.. there may not be a ceph-mgr on this host

Reply:
ah, true.  it needs the mgr service but not on this host.